Alertの作成

caramelribon opened the pull request:
エラー発生

caramelribon committed:
Alertの作成

エラー発生

caramelribon committed:
Alertの実装

様子見たいので一旦commitします

caramelribon committed:
Alertの実装

お手本通りにAlert完成

caramelribon committed:
Alertの実装

余分なコメントアウトを削除

caramelribon commented:
## Alertの実装

お手本通りにAlertが発生するように実装しました

- [x] お手本通りの見た目であるか　ホバー時の挙動、レスポンシブ対応など要確認
- [x] ブランチ名、ファイル名、クラス名、インデントなど、コーディング規則を逸脱していないか
- [x] 記述の重複はないか

GitHubPagesのページのリンク
[リンク](https://caramelribon.github.io/wakana-tatsuta-frontend-basic-2022-todoapp/)

MoAI522 commented on src/style.css:
```diff
@@ -6,3 +6,15 @@
+  transition: all 0.5s ease 0s;
+  transform: translateY(-30px);
+  opacity: 0;
+}
```
これ何のことか分かりづらくなるのでAtoms/Alert/index.jsxの方に書きましょう。
styledの中はsassみたいにセレクタの入れ子ができるので活用してください。

MoAI522 reviewed (commented):
めちゃくちゃ綺麗なコードで素晴らしいです。一点だけ修正お願いします。

caramelribon committed:
Alertの作成

・classをstyledの中に入れる

.fadeInと.fadeOutのクラスをわかりやすくするためAlert/index.jsxの中の、一番外の要素StyledAlertを作ってその中に入れ子としていれました

それにともない./style.cssのfadeInとfadeOutと, Alert/index.jsxでimportしていた所を削除しました

caramelribon commented:
## Alertの実装2nd

ご指摘頂いたところを修正しました。確認お願いします。

- [x] お手本通りの見た目であるか　ホバー時の挙動、レスポンシブ対応など要確認
- [x] ブランチ名、ファイル名、クラス名、インデントなど、コーディング規則を逸脱していないか
- [x] 記述の重複はないか

GitHubPagesのページのリンク
[リンク](https://caramelribon.github.io/wakana-tatsuta-frontend-basic-2022-todoapp/)

MoAI522 reviewed (approved):
LGTM
お疲れ様でした！
